restaurantPage.js

```diff
@@ -0,0 +1,96 @@
+fetch("all.json")
+  .then(response => response.json())
+  .then(jso => {
+    restaurantList = jso;
+    if (restaurantList.includes(window.location.hash.slice(1))) {
+      let restaurantName = window.location.hash.slice(1);
+      fetch(`${restaurantName}.json`)
+        .then(response => response.json())
+        .then(jso => {
+          createRestaurantPage(jso);
+        });
+    } else {
+      console.log("404");
+    }
+  });
+
+
+function createRestaurantPage(restaurantInfo) {
+  let element = document.getElementById("allInfo");
+  let name = document.createElement("h1");
+  let address = document.createElement("p");
+  let phoneLabel = document.createElement("h4");
+  let phone = document.createElement("a");
+  let hoursLabel = document.createElement("h4");
+  let hours = document.createElement("a");
+  let websiteLabel = document.createElement("h4");
+  let website = document.createElement("a");
+  let noteLabel = document.createElement("h3");
+  let notes = document.createElement("ul");
+  
+  document.querySelector('title').textContent = restaurantInfo.name;
+  name.textContent = restaurantInfo.name;
+  address.textContent = restaurantInfo.address;
+  phoneLabel.textContent = "Phone number : ";
+  phone.textContent = restaurantInfo.phone;
+  phone.href = `tel:${restaurantInfo.phone}`;
+  hoursLabel.textContent = "Hours : ";
+  if (restaurantInfo.hours !== undefined) {
+    hours.textContent = restaurantInfo.hours;
+  } else {
+    hours.textContent = "We show up whenever we feel like it!";
+  }
+  websiteLabel.textContent = "Website : ";
+  if (restaurantInfo.website !== undefined) {
+    website.textContent = restaurantInfo.website;
+    website.href = restaurantInfo.website;
+  } else {
+    website.textContent = "Website coming soon!";
+  }
+  noteLabel.textContent = "Notes : ";
+
+  restaurantInfo.notes.forEach(note => {
+    let listItem = document.createElement("li");
+    notes.appendChild(listItem);
+    note = marked(note);
```

Comment:
This statement is calling an undefined function. Which is probably why your restaurants page isn't displaying correctly.